openapi/backend/openapi.yaml

```diff
@@ -757,18 +757,26 @@ components:
         baseline:
           $ref: '#/components/schemas/BaselineInfo'
         usage:
-          type: number
-          format: float
-          description: >
-            Latest snapshot of the usage metric for the given feature.
-          minimum: 0.0
-          maximum: 100.0
+          type: object
+          properties:
+            chromium:
+              $ref: '#/components/schemas/UsageInfo'
         wpt:
           $ref: '#/components/schemas/FeatureWPTSnapshots'
       required:
         - feature_id
         - name
         - baseline_status
+    UsageInfo:
```

Comment:
Change the name to `ChromiumUsageInfo`. Not sure about the format others providers will have.